src/tests/utils/testUtilities.ts

```diff
@@ -5,8 +5,8 @@
 
 import { getLanguageService, JSONDocument, TextDocument } from '../../service'
 
-export function toDocument(text: string): { textDoc: TextDocument, jsonDoc: JSONDocument } {
-    const textDoc = TextDocument.create('foo://bar/file.asl', 'json', 0, text);
+export function toDocument(text: string, isYaml?: boolean): { textDoc: TextDocument, jsonDoc: JSONDocument } {
+    const textDoc = TextDocument.create(`foo://bar/file.${isYaml ? 'asl.yaml' : 'asl'}`, isYaml ? 'asl-yaml' : 'asl', 0, text);
```

Comment:
Are we renaming the language id from json to asl?

Reply:
It was always asl. It doesn't change anything for the tests though. Just fixed it for correctness.